### PR TITLE
Aws gov yaml changes

### PR DESCRIPTION
**Pull request by Randy424**
regarding: https://github.com/open-cluster-management/backlog/issues/15998
UI additions for AWS gov
Screen Shot 2021-09-28 at 4 54 49 PM

### CODE REVIEW

**Comment by Randy424** [comment]
@cadawson This PR could benefit from a preemptive review from doc team! If you have the time feel free to take a look. See the screenshots below for context.

Screen Shot 2021-09-28 at 4 54 54 PM


Screen Shot 2021-09-28 at 4 55 02 PM

File: frontend/public/locales/en/create.json [hunk 1]
```diff
@@ -256,7 +256,24 @@
...
     "creation.ocp.cluster.ost.external.dns": "External DNS IP addresses",
     "creation.ocp.cluster.ost.external.dns.placeholder": "Enter external DNS IP addresses",
-    "placeholder.creation.ocp.cluster.ost.external.dns": "OPTIONAL: Enter one or more external DNS IP addresses",
+    "placeholder.creation.ocp.cluster.ost.external.dns": "(Optional) Enter one or more external DNS IP addresses",
+    "creation.aws.privateAWS.info": "(Optional) Configuration for private AWS cluster provision",
+    "creation.aws.hostedZone.tooltip": "The ID of your existing Route 53 private hosted zone.",
+    "creation.aws.hostedZone.placeholder": "Enter hosted zone ID",
+    "creation.aws.ami.tooltip": "The ID of the AMI used to boot machines for the cluster. If set, the AMI must belong to the same region as the cluster.",
+    "creation.aws.ami.placeholder": "Enter amiID",
+    "creation.aws.subnet.subtitle": "Subets",
```
Comment (KevinFCormier) [inline frontend/public/locales/en/create.json:265]:
```suggestion
    "creation.aws.subnet.subtitle": "Subnets",
```

**Review by KevinFCormier — CHANGES_REQUESTED** [review]
Looks good! I just found one little typo.

**Pull request merged**